mailbag/controller.py

```diff
@@ -0,0 +1,19 @@
+
+
+class Controller:
+    """Controller - Main controller"""
+    
+    def __init__(self, args, formats):
+        self.args = args
+        self.format_map = {
```

Comment:
`format_map` here I think is essentially a hardcoded duplicate of what the registry in email_account.py is meant to do - remember, part of the goal is that formats should be dynamically loaded so that we can add them without having to keep an explicit list of them, and so that we can enable users to add/override new ones as plugins.

Rather than this, format_map could be a property that fetches EmailAccount.registry - i.e.

```python
@property
def format_map(self):
    return EmailAccount.registry
```